package.json

```diff
@@ -125,6 +124,7 @@
   },
   "dependencies": {
     "@floating-ui/dom": "^1.0.0",
+    "@types/react": "^18.2.17",
```

Comment:
Is this update really necessary? If not, please revert to `18.0.28` for consistency.

If it is, please let me know why, and also reinstall it as a dev dependency (`yarn add -D`)